gadget.yaml

```diff
@@ -1,7 +1,7 @@
 volumes:
   pi:
     schema: mbr
```

Comment:
gentle nudge to see if this can be gpt in time for uc22 :-)

Reply:
Theoretically it's a trivial change, but I need to make sure it actually works on all (recent) firmware versions. There was a slightly disturbing changelog entry in the recent rpi-eeprom bump (https://bugs.launchpad.net/ubuntu/+source/rpi-eeprom/+bug/1965539) "FAT/GPT fixes and file-system performance improvements" so I want to either make sure that it all works (on SD, eMMC, and USB/SSD boot) and/or stick something in the release notes to the effect of "you must have firmware later than YYYYMMDD installed on your EEPROM to boot this image".

Unfortunately I haven't had the time to dig into this yet (it'll probably require a full day of flashing EEPROM versions and testing).